python/neuroglancer/viewer_config_state.py

```diff
@@ -151,6 +151,7 @@ class ConfigState(JsonObjectWrapper):
     show_panel_borders = showPanelBorders = wrapped_property('showPanelBorders',
                                                              optional(bool, True))
     scale_bar_options = scaleBarOptions = wrapped_property('scaleBarOptions', ScaleBarOptions)
+    display_layer_item_value = displayLayerItemValue = wrapped_property('displayLayerItemValue', optional(bool, True))
```

Comment:
How about renaming this setting to show_layer_hover_values / showLayerHoverValues ?

Reply:
Yes, it is easier to understand what this is good for. I changed it, but now it is somewhat inconsistent with the naming in the layer_panel.css ...

Reply:
My original naming of the CSS classes was not so good.  If you want to fix that as well, that would be great, but I don't think it is that important.

Reply:
Since I cannot really tell where your naming is not so good, I'd better keep my hands off this...